cmd/kubeadm/app/apis/kubeadm/defaults.go

```diff
@@ -20,4 +20,6 @@ const (
 	DefaultServiceDNSDomain  = "cluster.local"
 	DefaultServicesSubnet    = "10.12.0.0/12"
 	DefaultKubernetesVersion = "v1.4.1"
+	DefaultAPIBindPort       = 6443
```

Comment:
I think the apiserver default has long been 6443 because it isn't a privileged port. We tend to override it to 443 on most installations so it'd be nice for the recommended configuration tool to use 443.